src/corefx/System.Globalization.Native/icushim.cpp

```diff
@@ -170,6 +170,13 @@ bool FindLibWithMajorMinorSubVersion(int* majorVer, int* minorVer, int* subVer)
     return false;
 }
 
+static int32_t g_icuPresent = 0;
+
+extern "C" int32_t GlobalizationNative_ICUPresent()
+{
+    return g_icuPresent;
```

Comment:
there is no easy way to access the config flag without doing pinvoke and passing the flag to system.Globalization.Native. I'll change this to have the  managed side specifically pinvoke to initialize the ICU if the flag is not set